content/applications/websites/ecommerce/shopper_experience.rst

```diff
@@ -7,7 +7,7 @@ Get paid
 .. toctree::
    :titlesonly:
 
-   shopper_experience/payment_acquirer
+   shopper_experience/payment_provider
```

Comment:
imo, let's change it to plural.
@Lau-Odoo Okay if we change this page to providerS plural?

If so, the folder and all image paths must be updated in the file as well

Reply:
No prob, we can change for providers